### PR TITLE
MTM-59277 Add RN for measurement series validation

### PR DESCRIPTION
**Pull request by karol-c8y**
RN for https://github.com/Cumulocity-IoT/cumulocity-core/pull/3030

### CODE REVIEW

**Review by jaro-herod — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    label: Core platform
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-59277
+version: 10.20.591.0
+---
+Property names used for fragment and series must not contain whitespaces nor the special characters (https://cumulocity.com/api/core/#operation/postMeasurementCollectionResource)
+Previously, `/measurement/measurements/series` endpoint accepted `series` query parameter which contains more than one dot, althought it returns empty response.
+With this change the endpoint returns `422` error if `series` query parameter contains more than one dot.
```
Comment (jaro-herod) [inline content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md]: I think you should mention that dots that are not fragment and series separator should be escaped. Right ?

File: content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - value: component-OG_650_b2
+    label: Core platform
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-59277
+version: 10.20.591.0
+---
+Property names used for fragment and series must not contain whitespaces nor the special characters (https://cumulocity.com/api/core/#operation/postMeasurementCollectionResource)
+Previously, `/measurement/measurements/series` endpoint accepted `series` query parameter which contains more than one dot, althought it returns empty response.
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md]:
```suggestion
Previously, the `/measurement/measurements/series` endpoint accepted a `series` query parameter which contained more than one dot, although it returned an empty response.
```

File: content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md (hunk 1 above)
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md]:
```suggestion
With this change the endpoint returns a `422` error if the `series` query parameter contains more than one dot.
```

File: content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+component:
+  - value: component-OG_650_b2
+    label: Core platform
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: MTM-59277
+version: 10.20.591.0
+---
+Property names used for fragment and series must not contain whitespaces nor the special characters (https://cumulocity.com/api/core/#operation/postMeasurementCollectionResource)
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md]:
```suggestion
Property names used for fragment and series must not contain whitespaces nor special characters (https://cumulocity.com/api/core/#operation/postMeasurementCollectionResource).
```

File: content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+date: 
+title: Measurement series endpoint returns error if series contains more than one dot
```
Comment (BeateRixen) [inline content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md]:
```suggestion
title: Measurement series endpoint returns an error if the series contains more than one dot
```

File: content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md (hunk 1 above)
Comment (karol-c8y) [reply content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md]: That's good question, do we want to admit that it's supported (when escaped) as an opposite to what https://cumulocity.com/api/core/#operation/postMeasurementCollectionResource says? 
Maybe we should document it here https://cumulocity.com/api/core/#operation/getMeasurementSeriesResource not in RN?

**Review by BeateRixen — APPROVED** [review]

**Review by pink-mongoose — APPROVED** [review]

File: content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md (hunk 1 above)
Comment (karol-c8y) [reply content/change-logs/platform-services/cumulocity-10-20-591-0-fail-series-validation-more-than-1-dot.md]: we discussed it and agreed not to share this information publicly

**Pull request merged**